test/server/core/ability/abilityTargets/CardTargetResolver.spec.ts

```diff
@@ -6,7 +6,7 @@ describe('Target resolvers for cards', function() {
                     phase: 'action',
                     player1: {
                         base: 'energy-conversion-lab',
-                        hand: ['reinforcement-walker', 'rebel-pathfinder', 'alliance-xwing', 'atst']
+                        hand: ['battlefield-marine', 'rebel-pathfinder', 'alliance-xwing', 'atst']
```

Comment:
Just curious, what happened with this test that broke it?

Reply:
Following up on the discussion from chat - just so we remember to come back to this, can you leave the original version in but commented out with a TODO about the bug?

Reply:
Just a bump on the above comment in case it got missed

Reply:
Done